packages/frontend/src/stockStrings.ts

```diff
@@ -20,7 +20,6 @@ export async function updateCoreStrings() {
   strings[C.DC_STR_VIDEO] = tx('video')
   strings[C.DC_STR_AUDIO] = tx('audio')
   strings[C.DC_STR_FILE] = tx('file')
-  strings[C.DC_STR_ENCRYPTEDMSG] = tx('encrypted_message')
```

Comment:
What error does this fix?

Reply:
It's removed from https://github.com/chatmail/core/blob/main/src/stock_str.rs

I just found it when checking translations. So yes, maybe should not part of this PR but should be removed anyway.
There are some more obsolete strings, so I will make another PR removing more of these

Reply:
`DC_STR_AEAP_EXPLANATION_AND_LINK` has _not_ been removed.